taqueria-flextesa-manager/lib.ts

```diff
@@ -47,7 +47,7 @@ export const startMininet = (parsedArgs: SanitizedArgs.t) => {
 		...getBootstrapFlags(parsedArgs),
 		'--until-level 200_000_000',
 		// TODO: Find a way of mapping protocol hash to protocol kind
-		`--protocol-kind "Alpha"`,
+		`--protocol-kind "Kathmandu"`,
```

Comment:
May I suggest we put it at the top and use a const variable?

e.g. `const PROTOCOL = "Kathmandu"`

Reply:
Sure, done